src/JTracker/Controller/AbstractAjaxController.php

```diff
@@ -79,6 +79,20 @@ public function execute()
 	}
 
 	/**
+	 * Allows setting the status header into the application
+	 *
+	 * @param   int  $code  The status code to set into the application
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function setHeader($code = 200)
```

Comment:
Bad method name IMO, implying you can set all the app headers when you've coded only support for the status header.